ldes-server-port-admin-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/rest/controllers/OpenApiEventStreamsController.java

```diff
@@ -0,0 +1,264 @@
+					"""))
+	})
+	List<EventStreamResponse> getEventStreams();
+
+	@Operation(summary = "Create or update an Event Stream based on the provided config")
+	@ApiResponse(responseCode = "200", content = {
+			@Content(mediaType = contentTypeTurtle, schema = @Schema(implementation = String.class), examples = @ExampleObject(value = """
+					@prefix ldes: <https://w3id.org/ldes#> .
+					@prefix custom: <http://example.org/> .
+					@prefix dcterms: <http://purl.org/dc/terms/> .
+					@prefix tree: <https://w3id.org/tree#>.
+					@prefix sh:   <http://www.w3.org/ns/shacl#> .
+					@prefix server: <http://localhost:8080/> .
+					@prefix xsd:  <http://www.w3.org/2001/XMLSchema#> .
+					@prefix example: <http://example.org/> .
+					@prefix parcels: <http://localhost:8080/parcels/> .
+
+					server:parcels a ldes:EventStream ;
+						ldes:timestampPath dcterms:created ;
+						ldes:versionOfPath dcterms:isVersionOf ;
+						custom:memberType <https://basisregisters.vlaanderen.be/implementatiemodel/gebouwenregister#Perceel> ;
+						custom:hasDefaultView "true"^^xsd:boolean ;
+						tree:shape parcels:shape .
+
+					parcels:shape a sh:NodeShape ;
+						sh:nodeShape [
+							sh:closed true ;
+							sh:propertyShape []
+							] ;
+						sh:deactivated true .
+					""")),
+			@Content(mediaType = contentTypeNQuads, schema = @Schema(implementation = String.class), examples = @ExampleObject(value = """
+					<http://localhost:8080/parcels> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://w3id.org/ldes#EventStream> .
+					<http://localhost:8080/parcels> <https://w3id.org/ldes#timestampPath> <http://purl.org/dc/terms/created> .
+					<http://localhost:8080/parcels> <https://w3id.org/ldes#versionOfPath> <http://purl.org/dc/terms/isVersionOf> .
+					<http://localhost:8080/parcels> <http://example.org/memberType> <https://basisregisters.vlaanderen.be/implementatiemodel/gebouwenregister#Perceel> .
+					<http://localhost:8080/parcels> <http://example.org/hasDefaultView> "true"^^<http://www.w3.org/2001/XMLSchema#boolean> .
+					<http://localhost:8080/parcels> <https://w3id.org/tree#shape> <http://localhost:8080/parcels/shape> .
+					<http://localhost:8080/parcels/shape> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/ns/shacl#NodeShape> .
+					<http://localhost:8080/parcels/shape> <http://www.w3.org/ns/shacl#nodeShape> _:genid1 .
+					<http://localhost:8080/parcels/shape> <http://www.w3.org/ns/shacl#deactivated> "true"^^<http://www.w3.org/2001/XMLSchema#boolean> .
+					_:genid1 <http://www.w3.org/ns/shacl#closed> "true"^^<http://www.w3.org/2001/XMLSchema#boolean> .
+					_:genid1 <http://www.w3.org/ns/shacl#propertyShape> _:genid2 .
+					""")),
+			@Content(mediaType = contentTypeJSONLD, schema = @Schema(implementation = String.class), examples = @ExampleObject(value = """
```

Comment:
The json will be properly formatted in the swagger UI if you remove this part:
`schema = @Schema(implementation = String.class),`